src/common/search-store.ts

```diff
@@ -29,7 +29,7 @@ export class SearchStore {
    * @param value Unescaped string
    */
   public static escapeRegex(value?: string): string {
-    return value ? value.replace(/[\-\[\]{}()*+?.,\\\^$|#\s]/g, "\\$&") : "";
+    return value ? value.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, "\\$&") : "";
```

Comment:
This looks wrong to me. Should `[` be escaped?
`[-\[\]{}()*+?.,\\^$|#\s]/g`

Reply:
No it doesn't need to be escaped because you cannot start another `one of` block within a `one of block`. This is the https://eslint.org/docs/rules/no-useless-escape rule